go/vt/vttablet/tabletmanager/action_agent.go

```diff
@@ -211,6 +217,8 @@ type ActionAgent struct {
 	_lockTablesTimer      *time.Timer
 	// unused
 	//_lockTablesTimeout    *time.Duration
+	// _isOnlineBackupRunning tells us whether there is a non-blocking backup that is currently running
```

Comment:
Can you remove the commented-out variable and `unused` comment above? It makes it look like everything below that point is unimportant. We also don't need to keep commented-out, unused code around since we have source control.